ql/src/java/org/apache/hadoop/hive/ql/exec/tez/TezProcessor.java

```diff
@@ -185,22 +184,21 @@ public void initialize() throws IOException {
     Configuration conf = TezUtils.createConfFromUserPayload(getContext().getUserPayload());
     this.jobConf = new JobConf(conf);
     this.jobConf.getCredentials().mergeAll(UserGroupInformation.getCurrentUser().getCredentials());
-    this.processorContext = getContext();
     initTezAttributes();
-    ExecutionContext execCtx = processorContext.getExecutionContext();
+    ExecutionContext execCtx = getContext().getExecutionContext();
     if (execCtx instanceof Hook) {
       ((Hook)execCtx).initializeHook(this);
     }
-    setupMRLegacyConfigs(processorContext);
+    setupMRLegacyConfigs(getContext());
     perfLogger.perfLogEnd(CLASS_NAME, PerfLogger.TEZ_INITIALIZE_PROCESSOR);
   }
 
 
   private void initTezAttributes() {
```

Comment:
maybe parametrize with `ProcessorContext processorContext` similar to `setupMRLegacyConfigs`

Reply:
hmm... The same context with setupMRLegacyConfigs. I think it is unnecessary to pass this parameter, and it wastes stack space...

Reply:
4/8 bytes to store the object ref?

Reply:
OK, I got it and I have fix.